src/api.ts

```diff
@@ -109,8 +109,7 @@ class RippleAPI extends EventEmitter {
   static _PRIVATE = {
     validate: validate,
     RangeSet,
-    ledgerUtils,
-    schemaValidator
```

Comment:
Since `schemaValidator` is no longer under `_PRIVATE`, you'll need to update [the reference in api-test.js](https://github.com/ripple/ripple-lib/blob/28796d37cb9af2c62d41f1f171aac82eafb480db/test/api-test.js#L16) in order for the unit tests to pass.

Reply:
Probably good to update the reference in broadcast-api-test.js too.